apps/frontend/src/libs/types/breadcrumb-item.type.ts

```diff
@@ -0,0 +1,9 @@
+import { type AppRoute } from "~/libs/enums/app-route.enum.js";
+import { type ValueOf } from "~/libs/types/types.js";
+
+type BreadcrumbItemType = {
```

Comment:
Let's rename it to BreadcrumbNavigationItem, and I think we won't need it in common types, so for now let's keep it in `src/libs/components/breadcrumbs/libs/types/breadcrumb-navigation-item.type.ts`
```suggestion
type BreadcrumbNavigationItem = {
```